icons/cc-info.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 30 30" xmlns="http://www.w3.org/2000/svg"><path d="M15 .469C6.975.469.469 6.978.469 15c0 8.027 6.506 14.531 14.531 14.531S29.531 23.027 29.531 15C29.531 6.978 23.025.469 15 .469zm0 6.445a2.46 2.46 0 110 4.922 2.46 2.46 0 010-4.922zm3.281 14.883a.703.703 0 01-.703.703h-5.156a.703.703 0 01-.703-.703V20.39c0-.389.315-.703.703-.703h.703v-3.75h-.703a.703.703 0 01-.703-.704v-1.406c0-.388.315-.703.703-.703h3.75c.388 0 .703.315.703.703v5.86h.703c.388 0 .703.314.703.703v1.406z" fill="currentColor"/></svg>
```

Comment:
This icon is probably part of the FontAwesome collection. Since CC may not own the icon, we should be cautious about including it in this project. Instead, we can import it directly from FontAwesome.